src/pages/navbar.tsx

```diff
@@ -1,12 +1,12 @@
 import Link from "next/link";
-import styles from '../styles/Nav.module.css'
 
 export default function Navbar() {
     return (
-            <nav className="navbar">
-                <Link href='/'>Home</Link>
-                <Link href='/login'>Login</Link>
-            </nav>
-       
+        <nav className="navbar">
+            <Link href='/'>Home</Link>
+            <Link href='/login'>Login</Link>
+            <Link href='/createUser'>Registration</Link>
```

Comment:
so what is it createUser or Registration - use a single name for an action otherwise it is getting confusing. Rename it to `signup` so it should look something like this:
`<Link href='/signUp'>Sign Up</Link>`
Rename on other places accordingly.